packages/jac/src/json-api.test.ts

```diff
@@ -26,7 +34,7 @@ describe('performs JsonApiClient request unit test', () => {
     test('should throw exception "baseUrl" argument not provided', () => {
       const api = new JsonApiClient({ baseUrl: VALID_BASE_URL_1 })
       expect(() => api.useBaseUrl('')).toThrow(
-        'Fetcher: invalid base URL. TypeError [ERR_INVALID_URL]: Invalid URL',
+        'Fetcher: invalid base URL. TypeError: Invalid URL',
```

Comment:
why did you change this?

Reply:
I had problems  with  this running tests on node 21.6.  Fixed by  using 18.19